base/hosts.c

```diff
@@ -1535,6 +1535,60 @@ gvm_hosts_exclude (gvm_hosts_t *hosts, const char *excluded_str)
   return gvm_hosts_exclude_with_max (hosts, excluded_str, 0);
 }
 
+/**
+ * @brief Creates a new gvm_host_t from the provided host_str. host_str can
```

Comment:
I'd like to keep "@brief" comments to one single line.  Anything more can go in the description below.

Here I would say just "from a host string".  The description of host_str would fit better in the "@param".